config/opal_functions.m4

```diff
@@ -399,6 +399,10 @@ AC_DEFUN([OPAL_FLAGS_UNIQ],[
                 opal_found=0
                 opal_i=`expr $opal_count + 1`
                 ;;
+        --param)
+                ompi_found=0
+                ompi_i=`expr $ompi_count + 1`
+                ;;
```

Comment:
My only comment is that it would be good to expand the comment on line 395 to include the URL to QuesarVII's post.  Otherwise looks fine to me.